manifests/bucketeer/charts/batch/templates/cronjob.yaml

```diff
@@ -0,0 +1,199 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "batch-server.fullname" . }}-experiment-status-updater
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ template "batch-server.name" . }}
+    chart: {{ template "batch-server.chart" . }}
+    release: {{ template "batch-server.fullname" . }}
+    heritage: {{ .Release.Service }}
+spec:
+  schedule: "*/30 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: experiment-status-updater
+              image: curlimages/curl:8.1.2
+              imagePullPolicy: IfNotPresent
+              command:
+                - "curl"
+              args:
+                - "-X"
+                - "POST"
+                - "-d"
+                - '{"job": "ExperimentStatusUpdater"}'
+                - "-H"
+                - "authorization: bearer {{ .Values.cronjob.token }}"
```

Comment:
Instead of using the new field `.Values.cronjob.token`, it seems better to use `{{ template "service-token-secret" . }}` as elsewhere for consistency.
[This PR](https://github.com/bucketeer-io/bucketeer/pull/453/files#diff-2a427e5e8f0b1a14e40f56488e9eb80dafdb58fec3335429b6018a473cc2b08fR28-R46) would be helpful.